addons/web/static/src/webclient/commands/command_provider_registry.js

```diff
@@ -0,0 +1,86 @@
+/** @odoo-module **/
```

Comment:
this file does not define the registry, it just add providers to it. I would rename it `command_providers.js`

Reply:
or `default_providers.js`